src/components/ReportActionItem/ReportPreview.js

```diff
@@ -115,23 +118,32 @@ function ReportPreview(props) {
                     accessibilityRole="button"
                     accessibilityLabel={props.translate('iou.viewDetails')}
                 >
-                    <View style={[styles.flexShrink1]}>
-                        {props.iouReport.hasOutstandingIOU ? (
-                            <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                {lodashGet(message, 'html', props.translate('iou.payerOwesAmount', {payer: managerName, amount: reportAmount}))}
-                            </Text>
-                        ) : (
-                            <View style={[styles.flexRow]}>
-                                <Text style={[styles.chatItemMessage, styles.cursorPointer, styles.colorMuted]}>
-                                    {lodashGet(message, 'html', props.translate('iou.payerSettled', {amount: reportAmount}))}
+                    <View style={styles.iouPreviewBox}>
+                        <View style={styles.flexRow}>
+                            <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
+                                <Text style={[styles.textLabelSupporting, styles.lh16]}>
+                                    {props.translate(props.iouReport.hasOutstandingIOU ? 'iou.payerOwes' : 'iou.payerPaid', {payer: managerName})}
                                 </Text>
                             </View>
-                        )}
+                            <Icon
+                                src={Expensicons.ArrowRight}
+                                fill={StyleUtils.getIconFillColor(getButtonState(props.isHovered))}
+                            />
+                        </View>
+                        <View style={styles.flexRow}>
+                            <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
+                                <Text style={styles.textHeadline}>{reportAmount}</Text>
+                                {!props.iouReport.hasOutstandingIOU && (
```

Comment:
Can we use [ReportUtils.isSettled](https://github.com/Expensify/App/blob/9d714850ee54fe040ed39eabfc4b2882d3733aa3/src/libs/ReportUtils.js#L235) here? I think that's more accurate since it actually checks the report state.

Reply:
That also prevents us from showing a checkmark when the report total is $0